prowler/providers/aws/services/rds/rds_instance_deprecated_engine_version/rds_instance_deprecated_engine_version.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_instance_deprecated_engine_version(Check):
+    def execute(self):
+        findings = []
+        available_versions = []
+
+        for instance in rds_client.db_instances:
+            available_versions.clear()
+            ##Filling the list of the available versions of the current engine
+            for iterate_version in rds_client.db_engines:
+                if instance.engine == iterate_version.engine:
+                    available_versions.append(iterate_version.engine_version)
+
+            report = Check_Report_AWS(self.metadata())
+            report.region = instance.region
```

Comment:
You must include `resource_arn` and `resource_tags` too to the `report`.

Reply:
Done!